docs-md/developer-guide/ksqldb-reference/show-queries.md

```diff
@@ -13,13 +13,15 @@ Synopsis
 --------
 
 ```sql
-SHOW QUERIES;
+SHOW | LIST QUERIES [EXTENDED];
 ```
 
 Description
 -----------
 
-List the running persistent queries.
+`SHOW QUERIES` lists queries running in the cluster.
+
+`SHOW QUERIES EXTENDED` lists queries running in the cluster in more detail.
 
 Example
 -------

```

Comment:
Adding in once we're satisfied with the final new output

Reply:
adding a sample cli output

Reply:
The example output in the description looks more like the JSON response from the server.  This looks good to me, except maybe I'd make the `state` field in the extended version match the output in the non-extended, i.e. 

```
"state": "RUNNING",
```

becomes:

```
"state": "RUNNING:2",
```

???

What I'm more interested in seeing though is the output from the CLI... The PR doesn't seem to have any new code to display this new info.  Does it need some, or will these new fields be output automagically?

Reply:
I actually changed it so that the state for a RunningQuery is set to the string representation of a QueryStateCount. I'm not sure the best way to change the state field for the show queries extended output. The QueryDescriptionFactory takes in a QueryMetadata and the ksqlHostQueryState Map and it populates the QueryDescription State field with the `QueryMetadata.getState()`.  I guess I could just modify the `forQueryMetadata()` function signature to also take in a String for the state but that seems a little excessive (?).

In regards to what actually shows up in the state field for extended, I think it should actually just be the string representation of the ksqlHostQueryState map and I get rid of the `Host Query Status` output in the CLI.